pinot-tools/src/main/java/org/apache/pinot/tools/service/PinotServiceManager.java

```diff
@@ -92,6 +92,8 @@ public String startRole(ServiceRole role, Map<String, Object> properties)
         return startBroker(new PinotConfiguration(properties));
       case SERVER:
         return startServer(new PinotConfiguration(properties));
+      case ADD_TABLE:
```

Comment:
This is not a pinot service. Perhaps you should pull out a base class, and add a `PinotSetupServiceManager` (I am sure there is a better name) in which you can add tables, ingest data, check for data being online, etc.?

Reply:
ServiceManagerRole perhaps? I currently don't have any other use case except setting up a table and prefer to not be too speculative.